modules/rosa-create-an-identity-based-policy.adoc

```diff
@@ -0,0 +1,166 @@
+                        "44.228.245.162/32",
+                        "44.238.205.35/32",
+                        "54.203.216.175/32",
+                        "34.237.49.153/32",
+                        "44.196.177.146/32",
+                        "52.23.117.40/32",
+                        "44.225.234.235/32",
+                        "44.241.225.78/32",
+                        "44.241.55.3/32",
+                        "34.237.180.56/32",
+                        "44.205.240.205/32",
+                        "52.54.93.238/32",
+                        "35.155.66.53/32",
+                        "44.231.249.237/32",
+                        "44.233.161.100/32",
+                        "3.229.185.234/32",
+                        "54.147.98.63/32",
+                        "54.163.100.197/32",
+                        "23.20.194.86/32",
+                        "23.22.242.238/32",
+                        "54.147.218.140/32",
+                        "52.21.229.141/32",
+                        "54.227.5.10/32",
+                        "54.146.138.135/32",
+                        "23.21.239.1/32",
+                        "52.20.145.130/32",
+                        "54.157.89.24/32",
+                        "107.22.162.110/32",
+                        "3.223.147.2/32",
+                        "54.88.225.66/32",
+                        "54.177.143.128/32",
+                        "54.219.250.189/32",
+                        "18.135.14.84/32",
+                        "18.135.218.119/32",
+                        "3.11.51.55/32",
+                        "3.233.86.181/32",
+                        "34.226.229.129/32",
+                        "44.194.44.138/32",
+                        "34.216.5.118/32",
+                        "52.11.52.9/32",
+                        "52.40.203.77/32",
+                        "18.217.173.123/32",
+                        "3.13.34.119/32",
+                        "3.19.160.232/32",
+                        "18.188.187.143/32",
+                        "18.216.245.132/32",
+                        "52.14.85.89/32",
+                        "52.21.184.148/32",
+                        "44.194.57.131/32",
+                        "18.188.65.148/32",
+                        "3.130.101.176/32",
+                        "3.130.198.233/32",
+                        "54.210.128.71/32",
+                        "54.227.100.14/32",
+                        "54.92.188.93/32",
+                        "107.22.5.187/32",
+                        "3.217.212.27/32",
+                        "52.22.56.3/32",
+                        "52.5.10.152/32",
+                        "54.237.41.201/32",
+                        "34.202.145.72/32",
+                        "52.205.239.95/32",
+                        "54.236.208.68/32",
+                        "3.234.64.191/32",
+                        "34.195.159.252/32",
+                        "34.228.34.122/32",
+                        "54.205.89.242"    
```

Comment:
This one is missing the /32. Sorry, I missed that in my earlier comment. The other three for the proxy IPs also appear to be missing from my other comments.
```
209.132.0.0/16
66.187.0.0/16
2620:0052:0004:0000:0000:0000:0000:0000/40
```